radsat-sk/operation/subsystems/RAntenna.c

```diff
@@ -1,7 +1,7 @@
 /**
  * @file RAntenna.c
  * @date Jan 15, 2022
- * @author Addi Amaya (caa746)
+ * @author Addi Amaya (caa746) and Atharva Kulkarni (iya789)
  */
 
 #include <RAntenna.h>
```

Comment:
Remove the `#include <hal/errors.h>` and replace with `#include <RCommon.h>` (if you're up to date with alpha this file should exist)